### ISSUE TITLE
Damage testing

### ISSUE THREAD

**Issue by Luciannuva** [popstarfreas/DG-TS#16]
One of the most inconvenient parts of DG pvp is that since the damage shown is not the damage delt you have to test damage against other players; So if no one is on, or you are fighting a good person, or if you can't aim you are going to have a hard time. it isn't too bad but what would make testing damage a lot easier is if you had a warp in the pvp dimention that was just a small room with 2 or 3 bot's. They don't need any ai or anything just the hitboxes. I say 2 or 3 because it would be nice to have one with a "tanky" loadout (like solar 3 stones and worm scarf) one with glass cannon armor (like nebula leggings dark artist's robe and chlorophyte headgear) and then a meta kit. That way you can see how you will fair against other builds. Overall it isn't "needed" it would just make things more convenient for everyone.

**Comment by Fraxo** [comment]
A warp with bots like the rofle bot in pve might not be the best idea, as that would confuse new players, but maybe target dummies that we could place somewhere near spawn or in the house or something that took the damage that modified weapons deal would be simpler and less confusing to new players. Either way, this is a good enhancement for PvP

**Comment by Luciannuva** [comment]
I mean... the reason it would be a warp is so that people don't just walk into it and go 'what the hell is this?" since it is a warp you would have to either read a sign or have someone tell you about it, but none of that really matters. The whole reason I suggested bots instead of dummies is because you told me dummies don't work blanc. also if you name the bots "Damage test Solar" I doubt anyone will be confused as to their purpose.

**Comment by Fraxo** [comment]
Yes, as of right now, target dummies do not work, I am bringing it up in case it is possible for Rofle to do so

**Comment by popstarfreas** [comment]
I think a patch for a bug that spammed updates for them is still around from a few Terraria updates ago that stopped the required updates for them to work. I believe it is not necessary anymore, so if target dummies are to be used it could work, but they are NPCs. The controller does not alter damage dealt to NPCs at this time.

**Comment by Luciannuva** [comment]
well I don't think it matters exactly how it is done. it is simply an object with a hitbox that displays damage values. whether as a bot or a dummy I am fine with whatever you choose to implement.

**Comment by popstarfreas** [comment]
When PvP gets the Special Item update, the display damage on weapon items will be accurate. Does this resolve this?

**Comment by Luciannuva** [comment]
For the most part. Although it would still help for identifying damage against different builds it wouldn't be half as important as when numbers aren't displayed properly. It also could be used to see if things like solar armor damage reduction actually works or not ect. But yeah, for the most part I think accurate numbers displayed would resolve the issue.
